fritz-demo-app/app/src/main/java/ai/fritz/heartbeat/Navigation.java

```diff
@@ -32,6 +33,11 @@ public static void goToLabelingActivity(Context context) {
         context.startActivity(labelActivity);
     }
 
+    public static void goToSkySegActivity(Context context) {
+        Intent skyActivity = new Intent(context, SkySegmentationActivity.class);
+        context.startActivity(skyActivity);
```

Comment:
Can you tell me what context refers to in this case (i.e what is the calling context being used for this intent)?